public/js/components/Tabs.js

```diff
@@ -10,12 +10,9 @@ const { debugTab } = require("../client");
 require("./Tabs.css");
 const dom = React.DOM;
 
-function Tabs({ tabs, newSource, paused, resumed,
-  selectTab, selectSource, loadSources }) {
-  function onClickTab(e) {
-    const tabActor = e.currentTarget.dataset.actorId;
-    debugTab({ tabActor, newSource, paused, resumed,
-               selectTab, selectSource, loadSources });
+function Tabs({ tabs, actions: boundActions }) {
+  function onSelect(tab) {
+    debugTab(tab.toJS(), boundActions);
```

Comment:
YES! I spent atleast an hour trying to think of a way to cleanup the actions list and it didn't occur to me you could pass them all in :) that makes me really happy to see.